src/Agent/NewRelic/Agent/Core/DataTransport/DataStreamingService.cs

```diff
@@ -762,11 +765,18 @@ private bool StreamRequests(PartitionedBlockingCollection<TRequest> collection,
 
                 _responseStreamsDic[consumerId] = new ResponseStreamWrapper<TResponse>(consumerId, responseStream, serviceAndStreamCancellationTokenSource.Token);
 
+                var hasUpdatedStreamingCount = false;
+
                 while (!serviceCancellationToken.IsCancellationRequested && _grpcWrapper.IsConnected)
                 {
-
                     if(!DequeueItems(collection, BatchSizeConfigValue, serviceCancellationToken, out var items))
                     {
+                        if(hasUpdatedStreamingCount)
+                        {
+                            Interlocked.Decrement(ref _countConsumersThatAreStreaming);
+                            LogMessage(LogLevel.Info, $"Count Consumers Actively Streaming {_countConsumersThatAreStreaming} [-1]");
```

Comment:
We probably don't want this log level to be "Info". I'm thinking that this might be better at the "Finest" level.